Sources/UIKit/AdaptableTextContainer.swift

```diff
@@ -146,20 +146,20 @@ extension UISegmentedControl {
 
     // `UISegmentedControl` has terrible generics ([NSObject: AnyObject]? or [AnyHashable: Any]?) on
     /// `titleTextAttributes`, so use a helper in Swift 3+
-    @nonobjc final func bon_titleTextAttributes(for state: UIControl.State) -> StyleAttributes {
-        let attributes = titleTextAttributes(for: state) ?? [:]
-        var result: StyleAttributes = [:]
-        for value in attributes {
-            #if swift(>=4.2)
-                result[value.key] = value
-            #else
-                guard let string = value.key as? StyleAttributes.Key else {
-                    fatalError("Can not convert key \(value.key) to String")
+    @nonobjc final func bon_titleTextAttributes(for state: UIControl.State) -> StyleAttributes? {
```

Comment:
This changes the public API, would it be possible keep this the same?

Reply:
Sure, I have reverted the changes to the signature of that method.

Reply:
I just realized that method is actually `internal`, so if we wanted to keep that change it wouldn't affect the public API. I'm fine with either direction.

Reply:
Also it looks like the tests are failing because of a warning. It's either because Zev's twitter account is unreachable or there is a placeholder team ID in the 'App' target.

```
 ** BUILD SUCCEEDED **
    
   Testing with `xcodebuild`. 
 -> BonMot (5.6.0)
    - WARN  | url: The URL (https://twitter.com/ZevEisenberg) is not reachable.
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | xcodebuild:  note: Building targets in parallel
    - NOTE  | xcodebuild:  note: Using codesigning identity override: -
    - NOTE  | xcodebuild:  note: Planning build
    - NOTE  | xcodebuild:  note: Constructing build description
    - NOTE  | xcodebuild:  warning: Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the App editor. (in target 'App' from project 'App')
    - NOTE  | xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file and one is not being generated automatically. (in target 'App' from project 'App')
    - NOTE  | xcodebuild:  note: Using codesigning identity override: 

[!] BonMot did not pass validation, due to 1 warning (but you can use `--allow-warnings` to ignore it).
You can use the `--no-clean` option to inspect any issue.

Exited with code exit status 1
CircleCI received exit code 1
```

Reply:
Might be related to #373 @ZevEisenberg